bin/fypp

```diff
@@ -2685,6 +2711,7 @@ class FyppOptions(optparse.Values):
         self.encoding = 'utf-8'
         self.create_parent_folder = False
         self.file_var_root = None
+        self.depfile = None
```

Comment:
Please add documentation to the `Attributes` section in the docstring of the class,